src/guides/v2.3/javascript-dev-guide/javascript/custom_js.md

```diff
@@ -92,6 +92,9 @@ Where the following notation is used:
 -  `<your_namespace>.<your_widget_name>` - the name of your custom [widget](https://glossary.magento.com/widget). According to the jQuery widgets naming convention, this value must contain a [namespace](https://glossary.magento.com/namespace) and name.
 -  `mage.<widget.name>` - the name of the Magento widget that you extend.
 
+{:.bs-callout-warning}
+When use custom js we recommend should minimal dependencies in your custom js. More extra dependencies will cause high load request make blocking render website. Dependencies default defined between square bracket by single quote as alias and seperated with comma. The more alias you defined the more time request from browser will make. Browser will load all dependencies follow order list
```

Comment:
```suggestion
When using custom JS, it is recommended that minimal dependencies should be used in custom JS. Additional dependencies will cause higher load requests which can block the rendering of the website. **Dependencies default defined between square bracket by single quote as alias and separated with comma**. The more aliases that are defined, the more requests the browser will make. **Browser will load all dependencies follow order list**
```
I've managed to correct the English for some of this, but the 2 sentences flagged as bold (**) need to be rewritten as they are very confusing and by correcting the text I may change their meaning. I would recommend that the "Dependencies ..." explanation also be accompanied by an example.